.changeset/few-tips-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': major
```

Comment:
Should be minor, unless its for svelte 5? 😅

Reply:
That's correct. Marking this as draft so it's not accidentally merged early.